### PR TITLE
Document git-po-helper and core-translation in po/README

### PR DESCRIPTION
**Pull request by jiangxin**
Add l10n conventions, usage of po helper program, and core translation in po/README.md

### CODE REVIEW

**Comment by jiangxin** [comment]
This pull request is based on commits from @bagasme . See PR #524

File: po/README.md [hunk 1]
```diff
@@ -340,4 +340,49 @@ Usage for git-po-helper:
...
+1. The subject of each l10n commit should be prefixed with "l10n: ".
+2. Do not use non-ASCII characters in the subject of a commit.
+3. The length of commit subject (first line of the commit log) should
+   be less than 50 characters, and the length of other lines of the
+   commit log should be no more than 72 characters.
+4. Add "Signed-off-by" trailer to your commit log, like other commits
+   in Git. You can automatically add the trailer by committing with
+   the following command:
+
+        git commit -s
```
Comment (gitster) [inline po/README.md:370]: Add a blank line here, just like the next item?

File: po/README.md [hunk 2]
```diff
@@ -159,6 +159,18 @@ in the po/ directory, where XX.po is the file you want to update.
...
+Fuzzy translation
+-----------------
+
+Fuzzy translation is a translation marked by comment "fuzzy" to let you
+know that the translation is out of date because the "msgid" has been
+changed. A fuzzy translation will be ignored when compiling using "msgfmt".
+Fuzzy translation can be marked by hands, but for most cases they are
+marked automatically when running "msgmerge" to update your "XX.po" file.
+
+After fixing the corresponding translation, you can remove the "fuzzy"
```
Comment (gitster) [inline po/README.md:171]: replace "can" with "must", I would think.

File: po/README.md (hunk 1 above)
Comment (jiangxin) [reply po/README.md:370]: fixed.

File: po/README.md (hunk 2 above)
Comment (jiangxin) [reply po/README.md]: fixed.

**Pull request merged**